frontend/src/lib/Utils.tsx

```diff
@@ -321,3 +321,27 @@ export function generateGcsConsoleUri(gcsUri: string): string | undefined {
 
   return GCS_CONSOLE_BASE + gcsUri.substring(GCS_URI_PREFIX.length);
 }
+
+const MINIO_URI_PREFIX = 'minio://';
+
+function generateArtifactUrl(source: string, bucket: string, key: string): string {
+  return `https:/artifacts/get?source=${source}&bucket=${bucket}&key=${key}`;
```

Comment:
I overlooked this url. I believe it should be `./artifacts/get?...` to be compatible with both kubeflow and standalone.
Because in kubeflow deployment. Pipelines is hosted at `/pipeline` path, if you use `/artifacts/get...`, then generated full path is `/artifacts/get..` instead of `/pipeline/artifacts/get...`.

Reply:
Made the URL relative.